docs/architecture/03-algorithm-equations.md

```diff
@@ -0,0 +1,114 @@
+# Equations for Score Calculation
+
+The following equations describe how the different levels of scores are calculated in the a11y-Score algorithm.
+
+## Scores
+
+### Individual Place Score
+
+Below is an example of how individual place scores are calculated for a specific criterion based on OSM tag values.
+$$ S_{\mathrm{p}} =  
```

Comment:
issue (blocking): I think, you're missing a line break here, the equation doesn't get rendered properly:

<img width="1029" height="133" alt="Image" src="https://github.com/user-attachments/assets/a2ce7681-34da-40ca-837b-6069f4597f26" />